docs/pages/guides/local-first.mdx

```diff
@@ -46,11 +48,26 @@ One way to think of local-first tools is to group them by the following categori
 - Fine-grained reactivity for minimal renders
 - Powerful sync and persistence (with Supabase support built-in)
 
-It works with Expo and React Native (via [React Native Async Storage](https://github.com/react-native-async-storage/async-storage?tab=readme-ov-file#react-native-async-storage)). This makes it a perfect match for building local-first mobile and web apps. Get started by using the [Legend-State Supabase example](https://github.com/expo/examples/tree/master/with-legend-state-supabase): `npx create-expo-app --example with-legend-state-supabase`.
+It works with Expo and React Native (via [React Native Async Storage](https://github.com/react-native-async-storage/async-storage?tab=readme-ov-file#react-native-async-storage)). This makes it a perfect match for building local-first mobile and web apps. Get started by using the [Legend-State Supabase example](https://github.com/expo/examples/tree/master/with-legend-state-supabase):
+
+```bash
```

Comment:
Let's use Terminal component here and for the rest of the commands: https://github.com/expo/expo/blob/main/docs/README.md#use-terminal-component-for-shell-commands-snippets